tests/network/utils.py

```
@@ -270,3 +270,18 @@ def get_destination_ip_address(destination_vm):
     assert dst_ip, f"Cannot get valid IP address from {destination_vm.name}."
 
     return dst_ip
+
+
+def basic_expose_command(
+    resource_name,
+    svc_name,
+    resource="vm",
+    port="27017",
+    target_port=SSH_PORT_22,
```

Flake8 (7.3.0)

[error] undefined name 'SSH_PORT_22'

(F821)